swagger/oauth.yaml

```diff
@@ -82,20 +87,32 @@ endpoints:
 
 definitions:
   bearer:
-    required:
-      - access_token
-      - scope
-      - token_type
-      - expires_in
     properties:
       access_token:
         type: "string"
+        description: "The access token issued by the authorization server."
+        example: "2YotnFZFEjr1zCsicMWpAA"
       refresh_token:
         type: "string"
+        description: "Refresh tokens are credentials used to obtain access tokens."
```

Comment:
...used to obtain new access tokens once authorization has already been granted.